lib/vector-api-client/src/test/mod.rs

```diff
@@ -21,15 +21,6 @@ pub struct ComponentLinksQuery;
 )]
 pub struct ErrorsTotalSubscription;
 
-/// Component errors totals subscription
```

Comment:
By instinct, I moved the subscription and related code away from this `vector-api-client/src/test/` directory and into the usual `vector-api-client/src/gql` directory. I'm not clear on why this `vector-api-client/src/test/` directory exists, so do let me know if there's something I'm missing. cc: @leebenson

Reply:
With CI passing, I'll go ahead and merge this, but happy to change it up if needed.

Reply:
Post-merge comment for posterity: `test` was an attempt at distinguishing the domain of each query, so it was clearer which queries are part of the API surface, and which exist solely for integration tests.

It's not necessary to have this separation, since attributes drive compilation. It's fine either way 👍🏻